ui/components/app/snaps/snap-ui-markdown/snap-ui-markdown.js

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ReactMarkdown from 'react-markdown';
 import {
-  TypographyVariant,
-  OVERFLOW_WRAP,
+  TextVariant,
+  OverflowWrap,
 } from '../../../../helpers/constants/design-system';
-import Typography from '../../../ui/typography/typography';
+import { Text } from '../../../component-library';
 
 const Paragraph = (props) => (
-  <Typography
+  <Text
     {...props}
-    variant={TypographyVariant.H6}
+    variant={TextVariant.bodySm}
```

Comment:
The font-size here is now too tiny.

cc @eriknson

Reply:
Font-size should be `bodyMd` (baseline, 14px)

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/19563/commits/20f698df12c88196235a936f332b4b0608af55f8